core/src/main/java/org/kiwix/kiwixmobile/core/utils/dialog/KiwixDialog.kt

```diff
@@ -227,6 +228,14 @@ sealed class KiwixDialog(
     getView = customGetView
   )
 
+  data class StartServer(val customGetView: (() -> View)?) : KiwixDialog(
+    R.string.progress_dialog_starting_server,
+    null,
+    null,
```

Comment:
If you are creating a null type to pass the null. We can pass an empty string here.

Reply:
@gouri-panda, We can not directly pass an empty string here as our positive message parameter takes an Int `val positiveMessage: Int` and we need to pass a valid string resource value to it. But we can make an empty string resource and pass it here.

Reply:
Done.